lib/logger.js

```diff
@@ -103,9 +105,37 @@ class Logger extends EventEmitter {
     let mac = undefined
     let ip = undefined
 
+    if (has(options, 'levels')) {
+      const type = typeOf(options.levels)
+      if (type !== 'array') {
+        const err = new TypeError('levels must be an array')
+        err.meta = {
+          got: type
+        }
+        throw err
+      }
```

Comment:
`Array.isArray` is probably the better thing to do here.

Reply:
yeah, that's what it started as, but I figured I could rely on `stdlib`.  I'll put it back.